src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+}
+
+.stars--1:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
The `nth-child` selector is used incorrectly here. The `.stars--1:nth-child(-n + 1)` selector will not apply the active star image correctly because it targets the first child of `.stars--1`, which is not the intended behavior. You should use `.stars--1 .stars__star:nth-child(-n + 1)` to apply the active image to the first star within the `.stars--1` container.